aws_codeseeder/resources/template.yaml

```diff
@@ -63,8 +63,8 @@ Resources:
     Type: AWS::IAM::ManagedPolicy
     Properties:
       Description: Managed Policy granting access to the AWS CodeSeeder resources
-      ManagedPolicyName: codeseeder-${seedkit_name}-${region}-resources
-      Path: /
+      ManagedPolicyName: !Sub codeseeder-${seedkit_name}-${AWS::Region}-resources
```

Comment:
The changes for the Account ID / Region are to avoid having to pass it into the template?

Doesn't the `${...}` need to now be replaced so it doesn't get picked up by the templating engine?

Reply:
Correct - those aren't necessary and can be resolved at deployment time. No the others don't have to be replaced.

Reply:
> No the others don't have to be replaced.

OK, looks like [`safe_substitute`](https://docs.python.org/3/library/string.html#string.Template.safe_substitute) says that this will be OK and replaced with the text. TIL. 👍